### PR TITLE
admin: Remove admin callback url param

### PR DESCRIPTION
**Pull request by jmarantz**
Commit Message: remove superfluous url argument from the args passed to admin callback.s That is available, if needed, from admin_stream.requestHeaders().getPathValue(). Generally it was only used for parsing query-params. Those should be obtained from admin_stream.queryParam() which will also parse query-params out of the body if they came in that way, e.g. from a POST.
Additional Description: This PR is large but IMO easy-to-review as it's almost entirely the removal of unreferenced function parameters.
Risk Level: low -- this is a non-functional change that just removes arguments from an API that is no longer used as of https://github.com/envoyproxy/envoy/pull/22424
Testing: //test/...
Docs Changes: n/a
Release Notes: n/a
Platform Specific Features: n/a

### CODE REVIEW

**Review by pradeepcrao — DISMISSED** [review]
LGTM

File: .bazelversion [hunk 1]
```diff
@@ -1 +1 @@
-6.0.0-pre.20220706.4
```
[the remark this answers is not in the document]
Comment (jmarantz) [reply .bazelversion]: Yes it was. Good catch! Done.

**Review by htuch — APPROVED** [review]
LGTM, thanks!

**Pull request merged**